docs/setup-cluster/deploy-cluster/installation-options.rst

```diff
@@ -51,9 +166,11 @@ Examples:
    # Set default Determined master address to example.org, port 8888.
    $ export DET_MASTER="example.org:8888"
 
-*********************************
- Step 3 - Set up Internet Access
-*********************************
+.. _internet-access:
+
+************************
+ Set Up Internet Access
+************************
```

Comment:
This feels like it belongs with certain agent installation information, not here in Installation Options.

Reply:
added a page to keep the content for #7763